src/main/java/org/folio/linked/data/service/resource/impl/ResourceServiceImpl.java

```diff
@@ -101,6 +101,8 @@ private Resource getResource(Long id) {
   private Resource saveNewResource(ResourceRequestDto resourceDto, Resource old) {
     var mapped = resourceDtoMapper.toEntity(resourceDto);
     metadataService.ensure(mapped, old.getFolioMetadata());
+    mapped.setCreatedDate(old.getCreatedDate());
+    mapped.setVersion(old.getVersion() + 1);
```

Comment:
And what about Create operation, don't we want to set createdBy there?

Reply:
it's out of the scope of the story, isn't it?

Reply:
I see it in the table in the ticket. But there is also statement: "The scope of this card is to add date fields to the resource table". So let's clarify with Doug, right? Otherwise there is a controversy and we can't say if we implemented the task or not.